Core/Object Arts/Dolphin/MVP/Views/Common Controls/UI.TabView.cls

```diff
@@ -89,7 +89,7 @@ calcClientRectangleFromRectangle: aRectangle
 calcRectangleFromClientRectangle: aRectangle 
 	"Private - Given a client rectangle represented by the <Rectangle> argument, answer the
 	window rectangle that would be required to achieve this taking into account the current
-	window styles etc Note that the answered rectangle is in the COORDINATE SYSTEM OF THE
+	window styles etc. Note that the answered rectangle is in the COORDINATE SYSTEM OF THE
```

Comment:
Not sure why this is shouting. Perhaps somebody got burned by that at the time, but I think we should lose the capitals.